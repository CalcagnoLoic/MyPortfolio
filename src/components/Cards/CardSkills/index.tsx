const Component = () => {
  return (
    <>
      <h2 id="skills" className="my-10 text-center font-[sofia] text-2xl md:text-4xl underline">
        My skills
      </h2>
    </>
  );
};

export default Component;
