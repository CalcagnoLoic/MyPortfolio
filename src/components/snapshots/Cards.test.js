import { Cards } from "../Cards";
import renderer from "react-test-renderer";

it("render correctly Cards components", () => {
    const componentsCards = renderer.create(<Cards />).toJSON();
    expect(componentsCards).toMatchSnapshot();
});
