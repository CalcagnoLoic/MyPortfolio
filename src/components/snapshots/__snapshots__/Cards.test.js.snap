// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Cards components 1`] = `
Array [
  <div
    className="group flex flex-col gap-5 md:gap-0 lg:gap-0 justify-center mt-7 rounded-xl overflow-hidden shadow-slate-700/50 shadow-lg"
  >
    <a
      href="https://calcagnoloic.github.io/Sunnyside-agency-react/"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="Projet"
        className="w-full cursor-pointer rounded-xl hover:scale-125 transition-all duration-700 hover:opacity-30"
        src="./img/sunnyside.png"
      />
    </a>
    <div
      className="absolute"
    >
      <div
        className="px-14 py-3 rounded-r-full bg-transparent group-hover:bg-slate-400 duration-700"
      >
        <a
          href="https://github.com/CalcagnoLoic/Sunnyside-agency-react/tree/main"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="Logo Github"
            className="w-16 opacity-0 group-hover:opacity-100 duration-700"
            src="./img/github.svg"
          />
        </a>
      </div>
    </div>
  </div>,
  <div
    className="group flex flex-col gap-5 md:gap-0 lg:gap-0 justify-center mt-7 rounded-xl overflow-hidden shadow-slate-700/50 shadow-lg"
  >
    <a
      href="https://calcagnoloic.github.io/loopstudios/"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="Projet"
        className="w-full cursor-pointer rounded-xl hover:scale-125 transition-all duration-700 hover:opacity-30"
        src="./img/loopstudio.png"
      />
    </a>
    <div
      className="absolute"
    >
      <div
        className="px-14 py-3 rounded-r-full bg-transparent group-hover:bg-slate-400 duration-700"
      >
        <a
          href="https://github.com/CalcagnoLoic/loopstudios"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="Logo Github"
            className="w-16 opacity-0 group-hover:opacity-100 duration-700"
            src="./img/github.svg"
          />
        </a>
      </div>
    </div>
  </div>,
  <div
    className="group flex flex-col gap-5 md:gap-0 lg:gap-0 justify-center mt-7 rounded-xl overflow-hidden shadow-slate-700/50 shadow-lg"
  >
    <a
      href="https://space-tourism-loic.netlify.app/"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="Projet"
        className="w-full cursor-pointer rounded-xl hover:scale-125 transition-all duration-700 hover:opacity-30"
        src="./img/spacetourism.png"
      />
    </a>
    <div
      className="absolute"
    >
      <div
        className="px-14 py-3 rounded-r-full bg-transparent group-hover:bg-slate-400 duration-700"
      >
        <a
          href="https://github.com/CalcagnoLoic/space-tourism"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="Logo Github"
            className="w-16 opacity-0 group-hover:opacity-100 duration-700"
            src="./img/github.svg"
          />
        </a>
      </div>
    </div>
  </div>,
]
`;
